tools/AutoTuner/src/autotuner/distributed.py

```diff
@@ -349,10 +392,14 @@ def parse_config(config, path=os.getcwd()):
             elif key == "_PINS_DISTANCE":
                 options += f' PLACE_PINS_ARGS="-min_distance {value}"'
             elif key == "_SYNTH_FLATTEN":
-                print('[WARNING TUN-0013] Non-flatten the designs are not '
+                print('[WARNING 0013] Non-flatten the designs are not '
```

Comment:
@luarss 
I thought we had agreed to keep the `TUN-` here.

Reply:
Added back.